marklogic-client-api/src/test/java/com/marklogic/client/test/RowManagerTest.java

```diff
@@ -1205,6 +1206,27 @@ public void testMapper() throws IOException, XPathExpressionException {
     }
   }
   @Test
+  public void testColumnInfo() throws IOException {
+    RowManager rowMgr = Common.client.newRowManager();
+
+    PlanBuilder p = rowMgr.newPlanBuilder();
+
+    PlanBuilder.ExportablePlan builtPlan =
+            p.fromView("opticUnitTest", "musician")
+                    .where(
+                            p.cts.andQuery(
+                                    p.cts.jsonPropertyWordQuery("instrument", "trumpet"),
+                                    p.cts.jsonPropertyWordQuery(p.xs.string("lastName"), p.xs.stringSeq("Armstrong", "Davis"))
+                            )
+                    )
+                    .orderBy(p.col("lastName"));
+
+    String result = rowMgr.columnInfo(builtPlan, new StringHandle()).get();
+    assertNotNull(result);
+    result = rowMgr.columnInfoAs(builtPlan, String.class);
+    assertNotNull(result);
```

Comment:
Should the assertions verify that the returned information identifies all of the expected columns and thus is useful?

To put it the other way, is it possible that an invalid response could be non-null?

Reply:
Thanks for the comment. I'll add more checking here.